app/lib/strip_whitespace.rb

```diff
@@ -0,0 +1,9 @@
+class StripWhitespace
+  def self.from_hash(hash)
+    hash.transform_values { |v| from_string(v) }
+  end
+
+  def self.from_string(string)
+    StripAttributes.strip_string(string, allow_empty: true)
```

Comment:
re: `allow_empty` - by default, the gem converts empty strings into nils. This broke a couple of tests, including https://github.com/DFE-Digital/apply-for-teacher-training/blob/647d54311fb24d9ef16f9bc19787f7228985b857/spec/system/vendor_api/vendor_receives_application_spec.rb#L30

wherein we expect empty address lines to be empty strings in the API response.

Given this, I've disabled the nullifying behaviour by setting `allow_empty: true`. Is that reasonable, or should we try and preserve the `'' → nil` convention?